cmd/admin-handlers.go

```diff
@@ -1759,8 +1758,97 @@ func (a adminAPIHandlers) ServerHardwareInfoHandler(w http.ResponseWriter, r *ht
 		// Reply with cpu network information (across nodes in a
 		// distributed setup) as json.
 		writeSuccessResponseJSON(w, jsonBytes)
-
 	default:
 		writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(ErrMethodNotAllowed), r.URL)
 	}
 }
+
+// KMSListKeysHandler - POST /minio/admin/v1/kms/key/list
```

Comment:
This is not v1 anymore

Reply:
Why? How can be *adding* a symbol (in that case a HTTP endpoint) a breaking change. Are you saying admin API does not follow semver but some other versioning?!

Reply:
Yes v1 is gone it's v2 now